packages/ember/addon/instance-initializers/sentry-performance.ts

```diff
@@ -145,6 +145,12 @@ export function _instrumentEmberRouter(
 
   routerService.on('routeWillChange', (transition: Transition) => {
     const { fromRoute, toRoute } = getTransitionInformation(transition, routerService);
+
+    // We want to ignore loading && error routes
+    if (transitionIsIntermediate(transition)) {
```

Comment:
Ember has a feature where you can easily add `loading` and `error` routes that are automatically picked up. However, if you do that it will also trigger transitions for them, which we don't really want here. So we need to filter them out here to avoid that we finish the "actual" routing spans too early.